### ISSUE TITLE
/angular2/http 404 not found

### ISSUE THREAD

**Issue by pdfarhad** [angular/angular-cli#685]
> I am facing a bug.

 1. angular-cli: 0.0.39
      node: 4.2.2
      os: win32 x64
2.  I am trying to use the ''angular2/http" in my services. It builds successfully but shows error in the console of Chrome browser.
3. Browser log 
zone.js:461 Unhandled Promise rejection: Error: XHR error (404 Not Found) loading http://localhost:4200/angular2/http
	    at XMLHttpRequest.desc.set.wrapFn [as _onreadystatechange] (http://localhost:4200/vendor/zone.js/dist/zone.js:769:30)
	    at ZoneDelegate.invokeTask (http://localhost:4200/vendor/zone.js/dist/zone.js:356:38)
	    at Zone.runTask (http://localhost:4200/vendor/zone.js/dist/zone.js:256:48)
	    at XMLHttpRequest.ZoneTask.invoke (http://localhost:4200/vendor/zone.js/dist/zone.js:423:34)
	Error loading http://localhost:4200/angular2/http as "angular2/http" from http://localhost:4200/app/js-tree.component.js ; Zone: <root> ; Task: Promise.then ; Value: Error: Error: XHR error (404 Not Found) loading http://localhost:4200/angular2/http
	    at XMLHttpRequest.desc.set.wrapFn [as _onreadystatechange] (http://localhost:4200/vendor/zone.js/dist/zone.js:769:30)
	    at ZoneDelegate.invokeTask (http://localhost:4200/vendor/zone.js/dist/zone.js:356:38)
	    at Zone.runTask (http://localhost:4200/vendor/zone.js/dist/zone.js:256:48)
	    at XMLHttpRequest.ZoneTask.invoke (http://localhost:4200/vendor/zone.js/dist/zone.js:423:34)
	Error loading http://localhost:4200/angular2/http as "angular2/http" from http://localhost:4200/app/js-tree.component.js

**Comment by zackarychapple** [comment]
Please update to beta.1 and regen the project.  You are using a pre-beta version. Once you do move over HTTP will be in `import { Http } from '@angular/http';`

**Comment by Codenator81** [comment]
@zackarychapple 
```
Alexs-iMac:my-app alex$ ng -v
angular-cli: 1.0.0-beta.1
node: 5.10.0
os: darwin x64
```
after create project in package.json not added 
```json
"@angular/http": "2.0.0-rc.1"
```
but it is mentioned at  system-config.ts
```ts
const barrels: string[] = [
  // Angular specific barrels.
  '@angular/core',
  '@angular/common',
  '@angular/compiler',
  '@angular/http',
  '@angular/router',
  '@angular/platform-browser',
  '@angular/platform-browser-dynamic',
```

**Comment by Brocco** [comment]
Closing, as this issue has been addressed via PR #635
